src/app/payments/container.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react'
+
+export default class Payments extends React.Component<any, any> {
```

Comment:
since we use typescript now we should define proptypes through interface

Reply:
yes, but for now we don't really know what type it will be. So I left it as any.